src/xcode/ENA/ENA/Source/ServerEnvironment/LocalServerEnvironment.swift

```diff
@@ -0,0 +1,66 @@
+// Local configuation
+
+import Foundation
+
+// MARK: - Map
+struct Map: Codable {
+	let serverEnvironments: [ServerEnvironment]
+
+	enum CodingKeys: String, CodingKey {
+		case serverEnvironments = "ServerEnvironments"
+	}
+}
+
+// MARK: - ServerEnvironment
+struct ServerEnvironment: Codable {
+	let name: String
+	let distributionURL, submissionURL, verificationURL: URL
+}
+
+struct LocalServerEnvironment {
+
+	struct Hosts {
+		let distributionURL: URL
+		let submissionURL: URL
+		let verificationURL: URL
+	}
+
+	private let environments: [ServerEnvironment]
+
+	init(bundle: Bundle = Bundle.main, resourceName: String = "ServerEnvironments") {
+		guard
+			let jsonURL = bundle.url(forResource: resourceName, withExtension: "json"),
+			let jsonData = try? Data(contentsOf: jsonURL),
+			let map = try? JSONDecoder().decode(Map.self, from: jsonData) else {
+
+			fatalError("Missing server environment.")
+		}
+
+		self.environments = map.serverEnvironments
+	}
+
+	func availableEnvironments() -> [ServerEnvironment] {
+		return environments
+	}
+
+	func loadServerEnvironment(_ name: String) -> ServerEnvironment {
+		guard let environment = availableEnvironments().first(where: { $0.name == name }) else {
+			fatalError("Missing server environment.")
+		}
+
+		return environment
+	}
+
+	func defaultEnvironment() -> ServerEnvironment {
+		return loadServerEnvironment("Default")
+	}
+
+	func getHosts(for environment: String) -> Hosts {
```

Comment:
This is is not Java! There is no need for a "get" because we don't fetch from servers or do any other asynchronous operation.

```suggestion
      func hosts(for environment: String) -> Hosts {
```

Reply:
removed. Its not used anyways.